src/aleph/vm/pool.py

```diff
@@ -83,20 +83,32 @@ async def create_a_vm(
             )
             self.executions[vm_hash] = execution
 
-        await execution.prepare()
-        vm_id = self.get_unique_vm_id()
+        # noinspection PyBroadException
```

Comment:
to avoid a warning in pycharm but I just found out if I replace it with an except Exception this also remove the warning so I will do that modification instead.

From my understanding `finally` would not make sense since we only want to forget the VM if it fail, not in all case when we exit the block